src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form form_big"
+      action="search"
+      data-qa="big"
+    >
+      <input
```

Comment:
The `placeholder` attribute should be more descriptive of the expected input. For example, 'Enter a city name like "Los Angeles"'.